data/json/items/classes/gun.json

```diff
@@ -100,7 +100,7 @@
     "//": "diminutive backup pistols that lack underbarrel rails, sight dovetails and decent places to affix stocks.",
     "valid_mod_locations": [
       [ "barrel", 1 ],
-      [ "belt_clip", 1 ],
+      [ "belt clip", 1 ],
```

Comment:
Does belt clip id use space instead of underscore?

Reply:
For mod locations name = id, and belt clip slot in game is named as `belt_clip`. I just removed the underscore